src/js/components/CheckBox/__tests__/__snapshots__/CheckBox-test.tsx.snap

```diff
@@ -225,10 +225,6 @@ exports[`CheckBox controlled 1`] = `
   -webkit-flex-direction: row;
   -ms-flex-direction: row;
   flex-direction: row;
-  -webkit-align-items: center;
```

Comment:
we need these styles to still get placed on the label so still need `labelProp` defined in checkbox.js

Reply:
reverted snapshots, please check now